CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### Current
 
+2017-01-14
+* Fixed: Return right parameter name in exception, #227, (@jeremysolarz)
```

Comment:
The changelog lists things by version, not date. Just add all your changes under the "current" section and once it's time to release a new version, I'll replace "current" with that JCommander version.